MongoDBAPI.py

```diff
@@ -1,70 +1,90 @@
+            db.create_collection(collection)
+        self.collection = db[collection]
 
     def read(self):
-        log.info('Reading All Data')
+        log.info('Reading all Articles')
         documents = self.collection.find()
         output = [{item: data[item] for item in data if item != '_id'} for data in documents]
         return output
 
     def write(self, data):
-        log.info('Writing Data')
-        new_document = data['Document']
-        response = self.collection.insert_one(new_document)
+        log.info('Writing one Article')
+        response = self.collection.insert_one({"title": data["title"], "content": data["content"]})
         output = {'Status': 'Successfully Inserted',
                   'Document_ID': str(response.inserted_id)}
         return output
 
+    def delete(self, articleID):
+        log.info('Deleting one Article')
+        response = self.collection.delete_one({"_id": ObjectId(articleID)})
+        output = {'DeletedCount': str(response.deleted_count)}
+        if response.deleted_count == 1:
+            output["Status"] = 'Successfully Deleted'
+        else:
+            output["Status"] = 'Could not delete object'
+        return output
+
 
 @app.route('/')
 def base():
     return Response(response=json.dumps({"Status": "UP"}),
-                    status=200,
+                    status=HTTPStatus.OK,
                     mimetype='application/json')
 
 
-@app.route('/mongodb', methods=['GET'])
-def mongo_read():
-    data = request.json
-    if data is None or data == {}:
-        return Response(response=json.dumps({"Error": "Please provide connection information"}),
-                        status=400,
-                        mimetype='application/json')
-    obj1 = MongoAPI(data)
+@app.route('/wiki', methods=['GET'])
+def wiki_readAllArticles():
+    obj1 = MongoAPI()
     response = obj1.read()
     return Response(response=json.dumps(response),
-                    status=200,
+                    status=HTTPStatus.OK,
                     mimetype='application/json')
 
 
-@app.route('/mongodb', methods=['POST'])
-def mongo_write():
+@app.route('/wiki', methods=['POST'])
+def wiki_createArticle():
     data = request.json
-    if data is None or data == {} or 'Document' not in data:
-        return Response(response=json.dumps({"Error": "Please provide connection information"}),
-                        status=400,
+    # expects "title" and "content" field in body
+    if 'title' not in data or 'content' not in data:
+        return Response(response=json.dumps({"Error": "Please provide article information (title and content)"}),
+                        status=HTTPStatus.BAD_REQUEST,
                         mimetype='application/json')
-    obj1 = MongoAPI(data)
+    obj1 = MongoAPI()
     response = obj1.write(data)
     return Response(response=json.dumps(response),
-                    status=200,
+                    status=HTTPStatus.OK,
```

Comment:
```suggestion
                    status=HTTPStatus.CREATED,
```

For creating a new ressource, return HTTP Status Code `201 - Created`.